mifosng-android/src/main/java/com/mifos/mifosxdroid/online/generatecollectionsheet/GenerateCollectionSheetPresenter.java

```diff
@@ -25,11 +44,19 @@
         extends BasePresenter<GenerateCollectionSheetMvpView> {
 
     private final DataManager mDataManager;
-    private Subscription mSubscription;
+    private CompositeSubscription mSubscription;
+    private DataManagerCollectionSheet collectionDataManager;
+    private Context c;
 
     @Inject
-    public GenerateCollectionSheetPresenter(DataManager dataManager) {
+    public GenerateCollectionSheetPresenter(DataManager dataManager,
```

Comment:
remove  if (mSubscription != null) mSubscription.unsubscribe(); from `detachView()` to mSubscription.clear().